src/tests/cpp/security/Security.cpp

```diff
@@ -41,16 +41,18 @@
 
 #include "security/CredentialGenerator.hpp"
 
-namespace FwkSecurity {
+#include <mutex>
+#include <util/concurrent/spinlock_mutex.hpp>
+
+namespace apache {
+namespace geode {
+namespace client {
+namespace testframework {
+namespace security {
```

Comment:
Probably but I didn't want to change their current scope without understanding the test. I wanted to just correct the warning around `using namespace`.